src/main/java/hng_java_boilerplate/util/JwtAuthenticationFilter.java

```diff
@@ -0,0 +1,63 @@
+package hng_java_boilerplate.util;
+
+import hng_java_boilerplate.user.serviceImpl.UserServiceImpl;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.NonNull;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.web.authentication.WebAuthenticationDetailsSource;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+
+
+@Component
+public class JwtAuthenticationFilter extends OncePerRequestFilter {
+
```

Comment:
Remove extra all extra spaces in this file

Reply:
Okay